example/delete-unused-gke-disks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
This seems like a very dangerous loaded gun to have sitting around as an example!!

Perhaps this script should be entirely commented out so a user has to edit it before (possibly accidentally) running it?

Reply:
Yes, I have moved it to https://github.com/lsst-dm/gcp-qserv.git, which contains various examples script for gcp.